configuring/proxies.rst

```diff
@@ -7,11 +7,13 @@ Here a few options that you can use to proxy your NodeBB forum.
     :hidden:
     :maxdepth: 0
 
+    Node.js<proxies/node>
```

Comment:
Can you place this after nginx, as we don't _officially_ support other proxies? :smile: